webapp/src/components/LinkedProfile/LinkedProfile.tsx

```diff
@@ -0,0 +1,24 @@
+import { Link } from 'react-router-dom'
+import { Profile } from 'decentraland-dapps/dist/containers'
+import { locations } from '../../modules/routing/locations'
+import { AssetType } from '../../modules/asset/types'
+import { VendorName } from '../../modules/vendor'
+import { Section } from '../../modules/vendor/decentraland'
+import { Props } from './LinkedProfile.types'
+
+export const LinkedProfile = (props: Props) => {
+  const { address, className } = props
+
+  return (
+    <Link
+      className={className}
+      to={locations.account(address, {
```

Comment:
I think you forgot to use the `browseOptions` here, I believe these are kind of the fallbacks and the props one should be the first option, right? 
```suggestion
      to={locations.account(address, browseOptions || {
```

Reply:
Oops, the defaults weren't needed at all, I've just removed them and used the `browseOption` prop directly.